pkg/gpio/gpio_linux.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package gpio provides functions for interacting with GPIO pins via the
+// GPIO Sysfs Interface for Userspace.
+package gpio
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+const gpioPath = "/sys/class/gpio"
+
+// Value represents the value of a gpio pin
+type Value bool
+
+// Gpio pin values can either be low (0) or high (1)
+const (
+	Low  Value = false
+	High Value = true
+)
+
+func (v Value) Dir() string {
+	if v == Low {
+		return "low"
+	}
+	return "high"
+}
+
+func (v Value) String() string {
+	if v == Low {
+		return "0"
```

Comment:
I feel like humans would appreciate something other than 0 or 1 reading the values? But idk

Reply:
it's complicated. Sometimes you want to print a whole bunch of GPIOs to form a 16 bit word, so the String() should probably be minimal I think. Also scripts sometimes might run programs that call these functions and if they get back 
some nice string with the value of 0 or 1
they have to disassemble that, whereas if they get back
0
or
1
it's easier to work with.
x=`gpio blah blah`
x will be 0 or 1.

Maybe we could have a function like this:
func Pretty(v ...Value) string {
print something people would like the concat of all these values
}
?

Reply:
I think I'm going to leave it at "0"/"1" for now since this is only the Value and doesn't give any information about the pin or anything, but I'll add in a pretty print with the pin number, etc. at some point :)